_issues/isnt-hillary-too-sick.md

```diff
@@ -6,34 +6,15 @@ categories: health
 
 Q: Isn't Hillary too sick to be President?
 
-A: No. While personal health is a consideration when choosing a
-President, Hillary **is a healthy woman**, who happens to have some mild
-health conditions.
+A: Hillary is a healthy woman.
```

Comment:
I would rather see a direct answer to the question: "No, Hillary is in excellent physical condition who eats well and exercises regularly." It can be shorter too, but putting "woman" in there seems odd as an answer.